PayPalCheckout/src/main/java/com/paypal/android/checkout/PayPalCheckoutListener.java

```diff
@@ -2,6 +2,9 @@
 
 import androidx.annotation.NonNull;
 
-public interface PayPalCheckoutResultCallback {
+/**
+ * Interface for receiving results of the PayPal payment flow.
+ */
+public interface PayPalCheckoutListener {
```

Comment:
We have some mis-aligned naming throughout this PR. We need to get consensus on what we want to call classes in this module, and the whole module (PayPal, PayPalCheckout ...)